glean-core/src/glean.udl

```diff
@@ -164,6 +165,12 @@ callback interface OnGleanEvents {
     void shutdown();
 };
 
+// A callback handler that receives the IDs of recorded events
+callback interface GleanEventListener {
+    // Called when an event is recorded, indicating the id of the event
+    void on_event_recorded(string id);
```

Comment:
I'd prefer either spelling out "identifier" or explaining that "id" means the category and name concatenated with a `.`.

We, alas, don't have specific nomenclature about how to talk about FQMNs (fully-qualified metric names) yet.